packages/snaps-controllers/src/cronjob/CronjobController.ts

```diff
@@ -302,6 +495,19 @@ export class CronjobController extends BaseController<
     });
   }
 
+  /**
+   * Generate a unique id for a background event.
+   *
+   * @returns An id.
+   */
+  private generateBackgroundEventId(): string {
+    const id = nanoid();
+    if (this.state.events[id]) {
+      this.generateBackgroundEventId();
```

Comment:
We expect the return values from `nanoid` to be unique and don't consider collisions